.github/ISSUE_TEMPLATE/1-bug_report.md

```diff
@@ -14,15 +14,15 @@ PROTIP: record your screen and attach it as a gif to showcase the issue.
 - How to record and attach gif: https://bit.ly/2Mi8T6K
 -->
 
-**What happened**:
+# What happened:
```

Comment:
Personally, I would change them to `##` for H2s.

I think the `#` H1 headings a bit too large and shouty, plus the issue title makes more sense  (semantically) as the H1.